js/src/chrome/data/chrome.data.js

```diff
@@ -1479,6 +1479,27 @@ Wu.Chrome.Data = Wu.Chrome.extend({
 		var allUsers = _.sortBy(_.toArray(app.Users), function (u) {
 			return u.store.firstName;
 		});
+		var itemsContainers = [];
+		var checkedUsers = {};
+
+		function onKeyUp(e) {
```

Comment:
not a big fan of functions inside functinos.. perhaps you could refactor to `this._onKeyUp` ?